pallets/settlement/src/benchmarking.rs

```diff
@@ -863,6 +863,13 @@ benchmarks! {
         }
     }
 
+    change_receipt_validity {
+        let signer = user::<T>("signer", 0);
+    }: _(signer.origin(), 0, false)
+    verify {
+        assert!(Module::<T>::receipts_used(&signer.account(), 0));
```

Comment:
You can use ensure here with an error message just in case something fails in compiled WASM.